serpextract/serpextract.py

```diff
@@ -432,15 +432,15 @@ def parse(self, url_parts):
 
                 # Now we have to check for a tricky case where it is a SERP but
                 # there are no keywords
-                if keyword is None and (u'&{}='.format(extractor) in query or
-                                        u'?{}='.format(extractor) in query):
+                if not keyword and (u'&{}='.format(extractor) in original_query or
```

Comment:
I'm fine with the `not keyword` change here, but I don't get why you've changed `query` to `original_query` here.